indico/modules/rb/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -8,951 +8,309 @@
 import roomURL from 'indico-url:rb.admin_room';
 import roomsURL from 'indico-url:rb.admin_rooms';
 import fetchRoomAttributesURL from 'indico-url:rb.admin_room_attributes';
-import fetchAttributesURL from 'indico-url:rb.admin_attributes';
 import fetchRoomAvailabilityURL from 'indico-url:rb.admin_room_availability';
-import fetchRoomEquipmentURL from 'indico-url:rb.admin_room_equipment';
+import fetchAttributesURL from 'indico-url:rb.admin_attributes';
 import updateRoomEquipmentURL from 'indico-url:rb.admin_update_room_equipment';
 import updateRoomAttributesURL from 'indico-url:rb.admin_update_room_attributes';
 import updateRoomAvailabilityURL from 'indico-url:rb.admin_update_room_availability';
 
-import _ from 'lodash';
-import React from 'react';
-import {bindActionCreators} from 'redux';
-import PropTypes from 'prop-types';
-import {connect} from 'react-redux';
+import {Form as FinalForm, FormSpy} from 'react-final-form';
 import {
   Button,
   Dimmer,
-  Dropdown,
   Form,
   Grid,
-  Header,
+  Label,
   Loader,
+  Menu,
   Message,
   Modal,
+  Tab,
 } from 'semantic-ui-react';
-import {Form as FinalForm, FormSpy} from 'react-final-form';
-import {FieldArray} from 'react-final-form-arrays';
+import React, {useEffect, useState, useCallback, useMemo} from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
 import arrayMutators from 'final-form-arrays';
-import shortid from 'shortid';
-import {indicoAxios, handleAxiosError} from 'indico/utils/axios';
-import {snakifyKeys, camelizeKeys} from 'indico/utils/case';
-import {
-  FieldCondition,
-  getChangedValues,
-  handleSubmitError,
-  FinalCheckbox,
-  FinalField,
-  FinalInput,
-  FinalRadio,
-  FinalTextArea,
-  parsers as p,
-} from 'indico/react/forms';
-import {FavoritesProvider} from 'indico/react/hooks';
+import {useDispatch, useSelector} from 'react-redux';
+import {snakifyKeys} from 'indico/utils/case';
 import {Translate} from 'indico/react/i18n';
-import {FinalEmailList, FinalPrincipal, ACLField} from 'indico/react/components';
+import {indicoAxios, handleAxiosError} from 'indico/utils/axios';
+
+import './RoomEditModal.module.scss';
```

Comment:
We usually have `scss` imports below all other imports. 

In general, you could reorganize the imports a bit (eg react import before SUIR). You can check the other components we have!

Same in other files of this PR.

Reply:
Agree, I did some re-organization based on the warnings I was getting. However, I would probably expect this to be linted, it is pretty hard to track.

Reply:
Is there any decent linter for this that works with eslint? The one we have isn't very powerful and just checks some basic things like relative imports after absolute ones...

Reply:
I did a quick research and couldn't find any solution for this particular case. However, I believe we can come back to this later in another task.